src/main/java/de/telekom/phonenumbernormalizer/numberplans/PhoneNumberValidationResult.java

```diff
@@ -0,0 +1,113 @@
+package de.telekom.phonenumbernormalizer.numberplans;
+
+import com.google.i18n.phonenumbers.PhoneNumberUtil.ValidationResult;
+
+/**
+ * Wrapper around the PhoneLib enum {@link ValidationResult} from Google
+ * <p>
+ * When the PhoneLib is validating a phone number it returns a value of the enum {@link ValidationResult}.
+ * </p><p>
+ * It differentiate two possible positive and five possible negative results. The value {@link ValidationResult#INVALID_LENGTH} for any negative case, which is not explicitly covered by any of the other four values.
```

Comment:
I'd suggest to apply a code formatter to limit lines to 120 chars - that would make it better readable on GitHub and also within IDEs.

Reply:
Corrected in Part 3